core/libp2p/security/plaintext/plaintext.cpp

```diff
@@ -12,10 +12,10 @@
 #include "libp2p/security/plaintext/plaintext_connection.hpp"
 
 #define PLAINTEXT_OUTCOME_TRY(name, res, conn, cb) \
-  auto name = (res);                               \
-  if (name.has_error()) {                          \
-    conn->close();                                 \
-    cb(name.error());                              \
+  auto(name) = (res);                              \
+  if ((name).has_error()) {                        \
+    (void)(conn)->close();                         \
```

Comment:
do not ignore result from `close`

Reply:
There's no way we can report about the close error, as we have already one error to be reported - `name.error()`. Also, it's not necessary, as the connection most probably gets abandoned after the error happened, so the destructor will be called, which will kill it anyway.